doc/documentation_workflow.rst

```diff
@@ -0,0 +1,152 @@
+Documentation workflow
+######################
+
+What you need to know
++++++++++++++++++++++
+
+We use **Sphinx** to generate documentation and **Read the Docs** to publish it.
```

Comment:
A matter of philosophy: should Sphinx and Read the Docs be links to the respective websites (top level)?

Reply:
Yes, that is a good idea! I have now linked Sphinx and Read the Docs. We should make sure such links remain up-to-date.

Reply:
I assume that the top-level URLs will remain stable over a very long time.

Reply:
That is true.